steps/server13.js

```diff
@@ -0,0 +1,28 @@
+// You should try to test if sendMessage function is working
```

Comment:
Once learners worked on `/public/script.js` to the point where the button should work, they can come back here to handle the `"newMessage"` event here. When they do, we could split this up into two or three shorter steps.